shm/inc/Cargo.hpp

```diff
@@ -2,17 +2,21 @@
 
 #include <string>
 
+constexpr size_t MAX_AMOUNT_OF_CARGO{ 1000 };
```

Comment:
This should be inside the class. Or if you don't need it in the header please move it to the anonymous namespace in .cpp file